packages/snowflake/src/lib/Snowflake.ts

```diff
@@ -106,6 +106,29 @@ export class Snowflake {
 	public timestampFrom(id: string | bigint): number {
 		return Number((BigInt(id) >> 22n) + this.#epoch);
 	}
+
+	/**
+	 * Returns a number indicating whether a reference snowflake comes before, or after, or is same as the given
+	 * snowflake in sort order.
+	 * @param a The first snowflake to compare.
+	 * @param b The second snowflake to compare.
+	 * @returns `-1` if `a` is older than `b`, `0` if `a` and `b` are equals, `1` if `a` is newer than `b`.
+	 * @example
+	 * ```typescript
+	 * const ids = ['737141877803057244', '1056191128120082432', '254360814063058944'];
+	 * console.log(ids.sort(a, b) => Snowflake.compare(a, b));
```

Comment:
```suggestion
	 * console.log(ids.sort(Snowflake.compare));
```

Reply:
ESLint whines about `this` being unbound, which is why I didn't write it like that. The solution, `this: void`, arises another ESLint error (invalid void) and doesn't silence the existing one.

Also, having the call done manually helps figuring out how to do DESC sort rather than ASC, which is basically negated calls.

Reply:
Doesn't take away that the old code is invalid JavaScript???

Reply:
I literally ran them right now, both examples work and produce the correct output as I wrote them in each `@example` block:

![image](https://user-images.githubusercontent.com/24852502/209539545-3bb61178-4487-46b3-870e-9b5b7be91c58.png)